src/atomate2/vasp/jobs/eos/base.py

```diff
@@ -0,0 +1,167 @@
+"""
+Module to define EOS jobs using the default atomate2 parameters.
+
+For consistency with atomate implementation, define EosRelaxMaker
+and EosDeformationMaker with legacy parameters
+
+Also define MP-compatible PBE-GGA jobs:
+    MPGGAEosRelaxMaker, MPGGADeformationMaker, and MPGGAEosStaticMaker;
+and MP-compatible r2SCAN meta-GGA jobs:
+    MPMetaGGAEosRelaxMaker, MPMetaGGADeformationMaker,
+    and MPMetaGGAEosStaticMaker
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from jobflow import job
+from pymatgen.analysis.eos import EOS, EOSError
+from pymatgen.transformations.standard_transformations import (
+    DeformStructureTransformation,
+)
+
+from atomate2.vasp.jobs.base import BaseVaspMaker, vasp_job
+from atomate2.vasp.sets.eos import EosSetGenerator
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from pymatgen.core import Structure
+
+    from atomate2.vasp.sets.base import VaspInputGenerator
+
+
+@job
+def postprocess_EOS(data_dict: dict, EOS_models: list | None = None):
+    """Take dict of E(V) data and fit to each equation of state in EOS_models."""
+    if EOS_models is None:
+        EOS_models = [
+            "murnaghan",
+            "birch",
+            "birch_murnaghan",
+            "pourier_tarantola",
+            "vinet",
+        ]
+
+    output = data_dict.copy()
+    for jobtype in ["relax", "static"]:
+        if len(list(data_dict.get(jobtype, []))) == 0:
+            continue
+        output[jobtype]["EOS"] = {}
+        for eos_name in EOS_models:
+            try:
+                eos = EOS(eos_name=eos_name).fit(
+                    data_dict[jobtype]["volumes"], data_dict[jobtype]["energies"]
+                )
+                output[jobtype]["EOS"][eos_name] = {
+                    **eos.results,
+                    "b0 GPa": float(eos.b0_GPa),
+                }
+            except EOSError:
+                output[jobtype]["EOS"][eos_name] = {}
+
+    return output
+
+
+@dataclass
+class DeformationMaker(BaseVaspMaker):
```

Comment:
See previous comment about removing `DeformationMaker` in line with the elastic workflow.